### ISSUE TITLE
Add DCE and other tools to Work section

### ISSUE THREAD

**Issue by replaid** [ocaorg/2018-redesign#31]
It will become the home of most of the former candidates for "icons on the bottom of the home page".

**Comment by replaid** [comment]
@raphaelisms Nice. Please remove the boilerplate content that comes before it.

**Comment by raphaelisms** [comment]
Boilerplate gone

**Comment by replaid** [comment]
Thanks. But the "Central Administration" bit is still boilerplate.

**Comment by replaid** [comment]
Also, while you're in there, please change occurrences of "goto" to "go to", since "goto" is considered harmful.

**Comment by raphaelisms** [comment]
The card styling is the new norm for landing pages that are essentially nav pages?

**Comment by replaid** [comment]
Yes, they were designed by SF and need their own markup (not just
copying the design on the existing site).


On Fri, Jun 8, 2018, at 5:22 AM, Raphael Platte wrote:
> The card styling is the new norm for landing pages that are
> essentially nav pages?> — You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub[1], or mute the
> thread[2].

Links:

  1. https://github.com/ocaorg/2018-redesign/issues/31#issuecomment-395703778
